### PR TITLE
Add ComboJS

### PR DESCRIPTION
**Pull request by combojs**

### CODE REVIEW

**Comment by tunnckoCore** [comment]
It seems it still isn't ready? Source code is mostly empty.

**Comment by combojs** [comment]
It shouldn't be. I'm using it in production at work.

The source code is a concatenation of two files:

1. Core - contains the render method and map method.

2. Component - extends Class and adds a method to update properties, as
well as a prop collection.

Please see instructions on its use here: combojs.com.

File: data.js [hunk 1]
```diff
@@ -8,6 +8,14 @@
 // See examples below.
 
 module.exports = [
+  {
+    name: "Combo.js",
+    github: "combojs/combo-js",
+    tags: ["component", "framework", "ui", "templating"],
+    description: "A lightweight library for building user interfaces",
+    url: "https://combojs.com/",
```
Comment (tunnckoCore) [inline data.js]: It seems it not support https, currently. Is it typo or you will get https soon?

**Comment by tunnckoCore** [comment]
> All changes are documented in CHANGELOG.md which is found on the root of the repo.

No matter of that, it totally not acceptable to have such commit history.

File: data.js (hunk 1 above)
Comment (tunnckoCore) [inline data.js]: Also it also seems that it not have redirect from www to non-www or the opposite. Just notes, nothing offensive.

Comment (combojs) [inline data.js]: Typo from copying an existing entry.

**Pull request closed without merging**